tests/CMakeLists.txt

```diff
@@ -48,3 +48,10 @@ add_test(
     NAME "machine-only-rules"
     COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${CMAKE_CURRENT_SOURCE_DIR}/test_machine_only_rules.py" --source_dir "${CMAKE_SOURCE_DIR}" --build_dir "${CMAKE_BINARY_DIR}"
 )
+
+if (SSG_BATS_TESTS_ENABLED AND BATS_EXECUTABLE)
+    add_test(
+        NAME "bash-unit-tests"
```

Comment:
Test name should be more specific, because if we are going to add test scenarios for other functions, this name will be misleading.

Reply:
I think it is okay to leave like, but instead of running only one test in the `COMMAND` option, run tests for a directory as described here: https://github.com/sstephenson/bats#test-suites. So basically it should be something like moving tests to a folder called `unit/bats/` and setting:
`COMMAND "${BATS_EXECUTABLE}" "${CMAKE_CURRENT_SOURCE_DIR}/unit/bats/"`

Reply:
Good point, I am changing the target to a directory.

Reply:
What do you think of moving the tests to a specific folder? if the amount of `bats` files increase the `tests` folder will be polluted. I suggest that you create a directory called `tests/unit/bats` and place the tests there.